### PR TITLE
ADEN-6114 | Introduce Aol OneMobile prebid adapter

### PR DESCRIPTION
**Pull request by kvas-damian**
https://wikia-inc.atlassian.net/browse/ADEN-6114

### CODE REVIEW

**Comment by WikiaTech** [comment]
SonarQube analysis reported 14 issues
* ![MAJOR][MAJOR] 13 major
* ![INFO][INFO] 1 info

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L19](https://github.com/Wikia/app/blob/ab2a13cb491e06cacaf400cbb072a53955cf629b/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L19): Remove the unused function parameter "$title". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L19](https://github.com/Wikia/app/blob/ab2a13cb491e06cacaf400cbb072a53955cf629b/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L19): Remove the unused function parameter "$article". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L19](https://github.com/Wikia/app/blob/ab2a13cb491e06cacaf400cbb072a53955cf629b/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L19): Remove the unused function parameter "$output". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L19](https://github.com/Wikia/app/blob/ab2a13cb491e06cacaf400cbb072a53955cf629b/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L19): Remove the unused function parameter "$user". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L19](https://github.com/Wikia/app/blob/ab2a13cb491e06cacaf400cbb072a53955cf629b/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L19): Remove the unused function parameter "$wiki". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L126](https://github.com/Wikia/app/blob/ab2a13cb491e06cacaf400cbb072a53955cf629b/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L126): Remove the unused function parameter "$scripts". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L127](https://github.com/Wikia/app/blob/ab2a13cb491e06cacaf400cbb072a53955cf629b/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L127): Refactor this code to rely on RequestContext instead of context-dependent globals. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AContextualWgVariableUsage)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L205](https://github.com/Wikia/app/blob/ab2a13cb491e06cacaf400cbb072a53955cf629b/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L205): Remove the unused function parameter "$skin". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L229](https://github.com/Wikia/app/blob/ab2a13cb491e06cacaf400cbb072a53955cf629b/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L229): Remove the unused function parameter "$jsExtensionPackages". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L229](https://github.com/Wikia/app/blob/ab2a13cb491e06cacaf400cbb072a53955cf629b/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L229): Remove the unused function parameter "$scssPackages". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)

[project-id]: wikia:app
[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Comment by WikiaTech** [comment]
SonarQube analysis reported 14 issues
* ![MAJOR][MAJOR] 13 major
* ![INFO][INFO] 1 info

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L19](https://github.com/Wikia/app/blob/a425dc022a2117a52f6b95acfca996957e4b64ad/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L19): Remove the unused function parameter "$title". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L19](https://github.com/Wikia/app/blob/a425dc022a2117a52f6b95acfca996957e4b64ad/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L19): Remove the unused function parameter "$article". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L19](https://github.com/Wikia/app/blob/a425dc022a2117a52f6b95acfca996957e4b64ad/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L19): Remove the unused function parameter "$output". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L19](https://github.com/Wikia/app/blob/a425dc022a2117a52f6b95acfca996957e4b64ad/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L19): Remove the unused function parameter "$user". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L19](https://github.com/Wikia/app/blob/a425dc022a2117a52f6b95acfca996957e4b64ad/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L19): Remove the unused function parameter "$wiki". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L126](https://github.com/Wikia/app/blob/a425dc022a2117a52f6b95acfca996957e4b64ad/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L126): Remove the unused function parameter "$scripts". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L127](https://github.com/Wikia/app/blob/a425dc022a2117a52f6b95acfca996957e4b64ad/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L127): Refactor this code to rely on RequestContext instead of context-dependent globals. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AContextualWgVariableUsage)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L205](https://github.com/Wikia/app/blob/a425dc022a2117a52f6b95acfca996957e4b64ad/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L205): Remove the unused function parameter "$skin". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L229](https://github.com/Wikia/app/blob/a425dc022a2117a52f6b95acfca996957e4b64ad/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L229): Remove the unused function parameter "$jsExtensionPackages". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L229](https://github.com/Wikia/app/blob/a425dc022a2117a52f6b95acfca996957e4b64ad/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L229): Remove the unused function parameter "$scssPackages". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)

[project-id]: wikia:app
[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

File: extensions/wikia/AdEngine/js/lookup/prebid/adaptersRegistry.js [hunk 1]
```diff
@@ -6,11 +6,13 @@ define('ext.wikia.adEngine.lookup.prebid.adaptersRegistry', [
 	'ext.wikia.adEngine.lookup.prebid.adapters.audienceNetwork',
 	'ext.wikia.adEngine.lookup.prebid.adapters.indexExchange',
 	'ext.wikia.adEngine.lookup.prebid.adapters.openx',
+	'ext.wikia.adEngine.lookup.prebid.adapters.onemobile',
 	'ext.wikia.adEngine.lookup.prebid.adapters.rubicon',
 	'ext.wikia.adEngine.lookup.prebid.adapters.rubiconDisplay',
 	'ext.wikia.adEngine.lookup.prebid.adapters.wikia',
 	'wikia.window'
-], function(aol, appnexus, appnexusAst, audienceNetwork, indexExchange, openx, rubicon, rubiconDisplay, wikia, win) {
+], function(aol, appnexus, appnexusAst, audienceNetwork, indexExchange, onemobile, openx, rubicon, rubiconDisplay,
```
Comment (fraszczakszymon) [inline extensions/wikia/AdEngine/js/lookup/prebid/adaptersRegistry.js]: Can we move all of them to new lines?

**Review by fraszczakszymon — APPROVED** [review]

**Comment by WikiaTech** [comment]
SonarQube analysis reported 14 issues
* ![MAJOR][MAJOR] 13 major
* ![INFO][INFO] 1 info

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L19](https://github.com/Wikia/app/blob/b753b03c6f7aa71a1dc4125744a91ecd15ca7a40/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L19): Remove the unused function parameter "$title". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L19](https://github.com/Wikia/app/blob/b753b03c6f7aa71a1dc4125744a91ecd15ca7a40/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L19): Remove the unused function parameter "$article". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L19](https://github.com/Wikia/app/blob/b753b03c6f7aa71a1dc4125744a91ecd15ca7a40/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L19): Remove the unused function parameter "$output". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L19](https://github.com/Wikia/app/blob/b753b03c6f7aa71a1dc4125744a91ecd15ca7a40/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L19): Remove the unused function parameter "$user". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L19](https://github.com/Wikia/app/blob/b753b03c6f7aa71a1dc4125744a91ecd15ca7a40/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L19): Remove the unused function parameter "$wiki". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L126](https://github.com/Wikia/app/blob/b753b03c6f7aa71a1dc4125744a91ecd15ca7a40/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L126): Remove the unused function parameter "$scripts". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L127](https://github.com/Wikia/app/blob/b753b03c6f7aa71a1dc4125744a91ecd15ca7a40/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L127): Refactor this code to rely on RequestContext instead of context-dependent globals. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AContextualWgVariableUsage)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L205](https://github.com/Wikia/app/blob/b753b03c6f7aa71a1dc4125744a91ecd15ca7a40/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L205): Remove the unused function parameter "$skin". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L229](https://github.com/Wikia/app/blob/b753b03c6f7aa71a1dc4125744a91ecd15ca7a40/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L229): Remove the unused function parameter "$jsExtensionPackages". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L229](https://github.com/Wikia/app/blob/b753b03c6f7aa71a1dc4125744a91ecd15ca7a40/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L229): Remove the unused function parameter "$scssPackages". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)

[project-id]: wikia:app
[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Pull request merged**